frontend/src/API.js

```diff
@@ -116,6 +117,30 @@ export const scoreResult = async ({
     }
 };
 
+export const scoreIntermediateResult = async ({
+    session,
+    participant,
+    result,
+}) => {
+    try {
+        const vars = {
+            session_id: session.id,
+            json_data: JSON.stringify(result),
+            csrfmiddlewaretoken: participant.csrf_token
+        };
+
+        const response = await axios.post(
+            API_BASE_URL + URLS.result.intermediateScore,
+            qs.stringify(vars)
```

Comment:
What is `qs`? Is it used to stringify json? Can't the built-in `JSON` library already do that?

Reply:
This is using the same pattern as other functions in `API.js`.

Reply:
Replacing `qs.stringify` by `JSON.stringify` doesn't work at any rate, tbh I also don't know why `qs.stringify` is needed here. See [this thread](https://stackoverflow.com/questions/15850866/are-json-and-node-js-querystring-the-same) for more info. Apparently Django expects a different form of serialization than provided by JSON.